### ISSUE TITLE
feat: add RAW support

### ISSUE THREAD

**Issue by yurijmikhalevich** [yurijmikhalevich/rclip#15]
- [ ] ARW
- [ ] DNG
- [ ] CRW
- [ ] CR2
- [ ] CR3

**Comment by yurijmikhalevich** [comment]
Closing because it's a very low priority. Let me know if you want this :)

**Comment by yurijmikhalevich** [comment]
Related discussion: https://github.com/yurijmikhalevich/rclip/discussions/87

**Comment by yurijmikhalevich** [comment]
Reopened and bumped priority because it is a popular user-requested feature.

**Comment by abidkhan484** [comment]
Could you please assign to this issue? @yurijmikhalevich

**Comment by yurijmikhalevich** [comment]
@abidkhan484, of course. Thank you for taking this one! Feel free to PR individual RAW format support in separate PRs so we can get them reviewed and merged sooner, and let me know if you'll have any questions! 😄

**Comment by abidkhan484** [comment]
Could you advise whether including raw formats in the IMAGE_REGEX constant in the main file is responsible for determining which files will be processed?

@yurijmikhalevich

**Comment by yurijmikhalevich** [comment]
@abidkhan484, we can do this, but `Pillow`, the image processing library we use, can't handle RAW files, so we will need to implement separate RAW file processors (or import a new library) to support new formats.

The logic should be:
- if there are RAW and JPG with the same name in the same directory, then process the JPG only and ignore the RAW file
- if there is only a RAW file, then process the RAW file

**Comment by yurijmikhalevich** [comment]
Dng added in https://github.com/yurijmikhalevich/rclip/pull/277 and released in 3.1.0 behind the `--experimental-raw-support` flag.
